### ISSUE TITLE
do we want to represent Grex?

### ISSUE THREAD

**Issue by mfrasca** [Ghini/ghini.desktop#260]
**ARTICLE 4: THE GREX**

INTERNATIONAL CODE OF NOMENCLATURE FOR CULTIVATED PLANTS
(ICNCP or Cultivated Plant Code)
incorporating the Rules and Recommendations for naming plants in cultivation
Eighth Edition

**Comment by RoDuth** [comment]
yes. Particularly for Orchids hybrids but also some other.  E.g.

_Paphiopedilum_ Jim Kie 'Springwater'

where Jim Kie is the grex name.  See http://www.aos.org/orchids/additional-resources/basics-of-orchid-names.aspx

_Paphiopedilum_ Jim Kie

would be all other members of this cross that are not clones of the 'Springwater' cultivar.

_Paphiopedilum_ 'Springwater'

would not normally ever be used.

**Comment by mfrasca** [comment]
my understanding is that you give a cultivar name to a species, be it well defined from a taxonomic point of view or not.  according to this logic, »_Paphiopedilum_ sp. 'Springwater'« would be a viable option and I have programmed it so in 3.2.  the [ICNCP](http://www.actahort.org/chronica/pdf/sh_10.pdf) document gives several examples similar to _Paphiopedilum_ 'Springwater' (e.g.: _Pelargonium_ 'Freak of Nature').
it all looks very specialistic, and possibly only interesting for a garden doing propagations (¿@TatiJaramilloV?), and selling or at least participating to expositions.  I was asking this question while still in Quito.  I think we can leave this noted and resting, for the time being.

**Comment by RoDuth** [comment]
You obviously have not dealt with orchid hybrids before.  It is specialised but in the same way that section and subsection is important to Rhododendron collectors grex is more important than cultivar to orchid collectors.  Just google orchid naming conventions

**Comment by mfrasca** [comment]
noted and resting.  I do not deal with plants, I deal with algorithms and data structures.  when needed.

**Comment by mfrasca** [comment]
> You obviously have not dealt with orchid hybrids before

you obviously make uninformed statements: I opened this issue precisely while in narrow contact with the JBQ, an institution using ghini precisely because of their orchid collection.  the question "do we want to represent Grex" was directed to them.

**Comment by RoDuth** [comment]
and their response?

@tmyersdn do you have much to do with orchids?  Maybe _Cymbidium_ down there?

I've been mentioning grex for orchid names since 2015 https://github.com/Bauble/bauble.classic/issues/179#issuecomment-160507281 It's not something we have much use for so no big deal for me but I know its a must for every hybrid orchid grower I've ever dealt with.  Just saying.

**Comment by mfrasca** [comment]
unfortunately, all their response is here in this very issue.
I don't think I can develop something useful for users with whom I'm not in direct contact.  the 2× two months in Quito were two important opportunities.  I'm trying to build a next one.
who knows we find someone interested in what you describe in https://github.com/Bauble/bauble.classic/issues/179#issuecomment-160507281
